github/Team.pyi

```diff
@@ -17,6 +18,7 @@ class Team(CompletableGithubObject):
         self, member: NamedUser, role: Union[str, _NotSetType] = ...
     ) -> None: ...
     def add_to_members(self, member: NamedUser) -> None: ...
+    def get_team_membership(self, member) -> Membership: ...
```

Comment:
You need to specify the type of member too.

Reply:
Add Union[str,NamedUser]